js/app.js

```diff
@@ -152,6 +152,11 @@ $(function(){
     }
 
   })
+  $('#tags-side-menu li').on('click', function() {
```

Comment:
This is the JS used for the tags gutter menu. As the tags menu is not being used, this is also unused. If we will never want tag gutter menu, this should be removed.